src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Presentation.Api.OData/WorkplacesController.g.cs

```diff
@@ -72,6 +72,7 @@ public async Task<ActionResult> GetRefToCountry([FromRoute] System.UInt32 key)
         return Ok(references);
     }
     
+    [HttpDelete("api/Workplaces/{key}/Country/{relatedKey}")]
```

Comment:
@anna-naumova25 the expectation is that the entity, and therefore the relation is deleted. Not just the relation!
Maybe we can still reuse this methos and send a different command!